docs/guides/great-api-docs.md

```diff
@@ -1,56 +1,58 @@
+- The **developers** who interact with the API to fulfill the use case. These developers can be internal or external and novice or experienced. Novice developers may require more hand-holding while experienced devs frequently use API docs as a reference. 
 
 Your developer experience must cater to both kinds of consumers. You want them to **decide and integrate with ease**. 
 
 #### API Reference
 
 API Reference Documentation has a couple of simple goals:
 
-- Help developers understand what is possible with an API.
+- Help developers understand what's possible with an API.
 
 - Show developers how to move from documentation to code.
 
 API Reference Documentation is the main purpose of the API Component (a.k.a the `elements-api` Web Component, or `import { API } from '@stoplight/elements';`).
 
-This documentation focuses on all the endpoints (a.k.a "operations") available in an API, and helps explain potential input and output values that can be in requests and responses. Some API reference documentation will just show an example, but it's more useful to expand upon that and explain not just what those values mean, but what other values could also be valid in various contexts.
+This documentation focuses on all the endpoints (a.k.a "operations") available in an API and helps explain potential input and output values that can be in requests and responses. Some API reference documentation only shows an example, but it's more useful to expand on that and explain not just what those values mean, but what other values could also be valid in various contexts.
+
+Elements groups them by tags and uses their summary in the sidebar, then shows off all the request and response information. For example, which security schemes are needed to talk to the endpoint, with example request code showing how users can try talking to it. It also shows important validations, such as which property must be a number, has a minimum value, or must be an email address.
 
-Elements groups them by tags and uses their summary in the sidebar, then shows off all the request and response information, like which security schemes might be needed to talk to the endpoint, with example request code showing how users can try talking to it. It will then also show important validations, like maybe a property has to be a number, has a minimum value, has to be an email address, or a date-time. 
 ### Sample Code
 
-Some folks have built their own API Client code, also known as Software Development Kits ("SDKs"). These are sometimes hand crafted, and sometimes automatically generated from OpenAPI too. 
+Some folks have built their own API Client code, also known as Software Development Kits ("SDKs"). These are sometimes hand-crafted, and sometimes automatically generated from OpenAPI too. 
```

Comment:
Last sentence sounds awkward to me, especially the 'too' at the end - I think this sounds better - These can be hand-crafted or automatically generated from OpenAPI.